schemas/com.mailgun/message_bounced/jsonschema/1-0-0

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message bounced event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_bounced",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "attachment-count": {
+      "description": "Number of attachments in attachment-x (where x is a number) fields.",
+      "type": "string",
+      "maxLength": 8
```

Comment:
So, can't we transform it to `integer` inside adapter?

Reply:
@chuwy I think everything comes to the adaptor as a string (This used to be an integer and the job would die before getting to the adaptor)

Reply:
> I think everything comes to the adaptor as a string

Right - but how it *leaves* the adaptor is at your discretion.

Reply:
@chuwy Thank you for you explanation yesterday, it helped a lot. Given my new understanding I would say that we should drop this field altogether. Agree?

Reply:
Hey @knservis do you mean `attachment-count`? If understand its purpose right - it has an analytical value, but obviously just not as a string, but as an integer.

Reply:
@chuwy This is the number of attachments on the message that was sent to the user. Methinks it is probably not useful for the same reason body-plain is not useful (i.e. you know what you sent out). I suppose it doesn't matter too much as it doesn't take up much space.

Reply:
Yep, I agree even if it's not useful - it's not a big deal as well (unlike `body-plain`). Let's leave.